src/auditwheel/main_repair.py

```diff
@@ -95,6 +95,17 @@ def configure_parser(sub_parsers):
         help="Do not check for higher policy compatibility",
         default=False,
     )
+    p.add_argument(
+        "--no-copy-site-libs",
+        dest="COPY_SITE_LIBS",
+        action="store_false",
+        help=(
+            "Do not copy libraries located in the site-packages directory from "
+            "other packages (update rpath only). The developer will need to ensure "
+            "that the skipped libraries are listed the package install dependencies."
+        ),
+        default=True,
+    )
```

Comment:
I would
- align the option name with the environment variable name, `--no-copy-site-libs` and `COPY_SITE_LIBS` is confusing
- set the default to be the previous behaviour, so to always copy site libs by default.

Reply:
I think the wrong default is what is causing tests to fail. The produced wheels in tests are not manylinux-compliant, so `auditwheel show` is setting the `linux_x86_64` tag rather than a `manylinux_2_17_x86_64` one.

Reply:
> align the option name with the environment variable name, `--no-copy-site-libs` and `COPY_SITE_LIBS` is confusing

I followed the design of `--no-update-tags` and `dest="UPDATE_TAGES"`:

https://github.com/pypa/auditwheel/blob/560509888bdb9cdfa98b41cc1641ec749b64b949/src/auditwheel/main_repair.py#L74-L83

Here `default=True` means `args.UPDATE_TAGES` is `True` if the option `--no-update-tags` is not present. If the option `--no-update-tags` present, `action="store_false"` changes `args.UPDATE_TAGES` to `False`.

> set the default to be the previous behaviour, so to always copy site libs by default.

`args.COPY_SITE_LIBS=True` is the default behavior.

Reply:
> `args.COPY_SITE_LIBS=True` is the default behavior.

OK, sorry for the noise. Something is still off since the tests are failing.